### PR TITLE
James/features/sti

### PR DESCRIPTION
**Pull request by villanuevajamesfvillanueva**
1. updated notes
2. implemented sti in users model
    a. Broker model and Buyer model inherits from User model
    b. Added new links to devise login form redirecting to Broker Sign up and Sign in pages
image

### CODE REVIEW

**Review by dlanoronald123 — APPROVED** [review]
All goods my friend

File: app/models/user.rb [hunk 1]
```diff
@@ -3,4 +3,12 @@ class User < ApplicationRecord
...
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
+
+  #   def broker?
+  #     type == 'Broker'
+  #   end
+
+  #   def buyer?
+  #     type == 'Buyer'
+  #   end
```
Comment (analyn-cajocson) [inline app/models/user.rb]: Remove unnecessary comemnts

File: config/routes.rb [hunk 2]
```diff
@@ -1,5 +1,13 @@
 Rails.application.routes.draw do
+    # devise_for :users, :buyers, :brokers
+
+    devise_for :buyers, :brokers
+
+    # devise_for :users, controllers: {
+    # registrations: 'buyers/registrations'
+    # }
```
Comment (analyn-cajocson) [inline config/routes.rb]: Remove unnecessary comments

**Review by analyn-cajocson — CHANGES_REQUESTED** [review]

File: app/models/user.rb (hunk 1 above)
Comment (villanuevajamesfvillanueva) [reply app/models/user.rb]: made some changes, removed unnecessary comments

File: config/routes.rb (hunk 2 above)
Comment (villanuevajamesfvillanueva) [reply config/routes.rb]: made some changes, removed unnecessary comments

**Review by analyn-cajocson — APPROVED** [review]

**Pull request merged**